addons/web_hierarchy/static/src/hierarchy_controller.xml

```diff
@@ -27,10 +27,8 @@
                     <t t-component="searchBarToggler.component" t-props="searchBarToggler.props"/>
                 </t>
                 <t t-set-slot="default" t-slot-scope="layout">
-                    <t t-if="displayNoContent">
-                        <t t-call="web.ActionHelper">
-                            <t t-set="noContentHelp" t-value="props.info.noContentHelp"/>
-                        </t>
+                    <t t-if="model.resIds.length === 0 and props.info.noContentHelp" t-call="web.ActionHelper">
```

Comment:
displayNoContent wasn't defined? There's another occurrence of the variable in the template (line 5)...

Reply:
A test would be nice :-)